refactor/results/app/core/db.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+import logging
+import os
+import sqlite3
+from typing import Optional, Tuple, Union
+
+
+class Database:
+    def __init__(self, results_db: str = "results.db"):
+        self.results_db = os.environ.get("RESULTS_DB") or results_db
```

Comment:
I think this should pull from config instead of env... see
https://github.com/AgnostiqHQ/covalent/pull/281/files#diff-f5a263f3dc58c29625b2c1e387958a2f197f3b7f3328cf7fdac3363ef4af615cR35

Reply:
Fixed